modules/gdpr_fields/plugins/relationships/entities_from_field.inc

```diff
@@ -0,0 +1,254 @@
+<?php
```

Comment:
@yanniboi This looks like it would be really useful elsewhere but not here if we're switching to properties. Is there any other need for this that I am missing?